Adding state_snapshot grpc stubs to Python Clientservice. These were missing from the Python ClientService class.

alfredd opened the pull request:
* **Problem Overview**  
state_snapshot grpc bindings were missing from the ClientService python class under Apollo tests. 

The changes include 
1. Adding a line to the update_python_bindings.py file that allows the grpc bindings for state_snapshot.proto to be generated. 
2. Generating and adding the grpc python bindings for changes to the .proto files. 
3. Adding code for the state_snapshot generated bindings in the csutil.py file.

shgandhi commented on tests/apollo/clientservice/csutil.py:
```diff
@@ -51,3 +54,16 @@ def subscribe(self, id, legacy=False):
+
+    def read_as_of(self, id, keys_list=[""]):
+        req = state_snapshot_proto.ReadAsOfRequest(snashot_id = id, keys = keys_list)
+        return self.state_snapshot_stub.ReadAsOf(req)
```
Please add newline at end of file.

teoparvanov reviewed (approved):
LGTM, thanks @alfredd ! You can merge this after addressing Shruti's comment about the commit messages.

alfredd commented:
> Thanks for the change @alfredd. LGTM. However, please change your commit messages according to the guidelines mentioned here - https://cbea.ms/git-commit/.

@shgandhi I've added a new line and updated my comment.

alfredd commented:
I'm closing this pull request as it has become unnecessarily convoluted. I will create another pull request with proper comments.
